### DISCUSSION TITLE
I cannot install this mod to the v0.25

### DISCUSSION THREAD

**Discussion by sergiocerrutti** [0ad-matters/community-maps-2#15] [General]
If a click on it, it says:
`Failed to get metadata_blob from modFile.`

Also I see it has no dependency of any version of the game as other mods.
Thanks in advance.

**Comment by andy5995** [comment]
Thanks for the report @sergiocerrutti .

What operating system are you using?

Are you installing using the .zip or the .pyromod file?

**Comment by andy5995** [comment]
I added some extra [info about installing](https://github.com/0ad-matters/community-maps-2#installing) to the README.
